Userland/Libraries/LibGfx/ICC/TagTypes.cpp

```diff
@@ -760,8 +760,14 @@ ErrorOr<NonnullRefPtr<NamedColor2TagData>> NamedColor2TagData::from_bytes(Readon
 
     auto& header = *bit_cast<NamedColorHeader const*>(bytes.data() + 8);
 
-    unsigned const record_byte_size = 32 + sizeof(u16) * (3 + header.number_of_device_coordinates_of_each_named_color);
-    if (bytes.size() < 2 * sizeof(u32) + sizeof(NamedColorHeader) + header.count_of_named_colors * record_byte_size)
+    Checked<u32> record_byte_size = 3;
+    record_byte_size += header.number_of_device_coordinates_of_each_named_color;
+    record_byte_size *= 32 + sizeof(u16);
```

Comment:
Shouldn't this be `*= sizeof(u16)` followed by a `+= 32`?

Reply:
Yes, I've checked the results are identical.